lib/ts/recipe/emailpassword/api/signin.ts

```diff
@@ -36,11 +37,13 @@ export default async function signInAPI(apiImplementation: APIInterface, options
     let result = await apiImplementation.signInPOST({ formFields, options, userContext: {} });
 
     if (result.status === "OK") {
+        debugLoggerWithCode[loggerCodes.API_RESPONSE]("SignInAPI", result.status);
```

Comment:
I'd prefer if both of these kinds of logs were done in a single place (`supertokens.ts` or `recipeModule.ts`). A few more things:
- We should log errors (e.g., 500) if we log normal responses.
- How does this interact with user overrides?
- If the API is disabled, what do we log? Maybe add a separate log for that.

Reply:
> I'd prefer if both of these kinds of logs were done in a single place (supertokens.ts or recipeModule.ts)

Sure

> - We should log errors (e.g., 500) if we log normal responses.
> - If the API is disabled, what do we log? Maybe add a separate log for that.

Sure, the logs that have been added are only example logs. The idea was that over time we would add more logs